src/styles/style.css

```diff
@@ -0,0 +1,5 @@
+@import 'blocks/stars.css';
```

Comment:
Be cautious when using `@import` in CSS as it can lead to performance issues since imported files are downloaded in sequence, not in parallel. Consider combining CSS files during the build process or using link tags in HTML for parallel downloads.